Python/Setup/swix/Packages/dirs.proj

```diff
@@ -0,0 +1,36 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\SetupProjectBefore.settings" />
+
+  <ItemDefinitionGroup>
+    <!--
+    We explicitly disable signing for individual projects and batch
+    sign all outputs in one go.
+    -->
+    <ProjectFile>
+      <Properties>SignType=none</Properties>
+    </ProjectFile>
+  </ItemDefinitionGroup>
+
+  <PropertyGroup>
+    <StopOnFirstFailure>true</StopOnFirstFailure>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectFile Include="Microsoft.PythonTools.Core.Vsix.swixproj"/>
```

Comment:
No, they are part of this one, the Django one, and the Uwp one.

---
In reply to: [69642498](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69642498) [](ancestors = 69642498)